CHEWBBACA/CHEWBBACA_NS/stats_requests.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+AUTHORS
+
+    Mickael Silva
+    github: @
+
+    Pedro Cerqueira
+    github: @pedrorvc
+
+    Rafael Mamede
+    github: @rfm-targa
+
+DESCRIPTION
+
+"""
+
+
+import argparse
+import concurrent.futures
+from getpass import getpass
+
+from SPARQLWrapper import SPARQLWrapper
+
+try:
+    from utils import auxiliary_functions as aux
+except:
+    from CHEWBBACA.utils import auxiliary_functions as aux
+
+
+# virtuoso_server = SPARQLWrapper(app.config['LOCAL_SPARQL'])
```

Comment:
this should be cleaned up too?

Reply:
Updated file and removed commented code.